com.bardsoftware.constable/src/main/kotlin/com/bardsoftware/constable/H2Manager.kt

```diff
@@ -0,0 +1,24 @@
+package com.bardsoftware.constable
+
+import org.h2.tools.RunScript
+import java.io.FileReader
+import java.sql.DriverManager
+import java.sql.SQLException
+import java.util.logging.Level
+import java.util.logging.Logger
+
+class H2Manager {
+    companion object {
+        private val logger = Logger.getLogger(H2Manager::class.java.name)
```

Comment:
You can just move this logger to the file level.

Reply:
Done